xml/obs_build_job_constraints.xml

```diff
@@ -3,276 +3,462 @@
+    <para> Memory specific. </para>
+    <sect4>
+     <title><literal>size</literal></title>
+     <para> To require a minimal memory size. Swap space is not taken into
+      account here. </para>
+     <para> Example for _constraints file:</para>
+      <screen>&lt;constraints&gt;
+   &lt;hardware&gt;
+     &lt;physicalmemory&gt;
+       &lt;size unit=M&gt;1400&lt;/size&gt;
+     &lt;/physicalmemory&gt;
+   &lt;/hardware&gt;
+&lt;/constraints&gt;</screen>
+      <para>Example for project configuration: </para>
+      <screen>Constraint: hardware:physicalmemory:size unit=M 1400</screen>
+      <para>Example for RPM spec file or Dockerfile:</para>
+      <screen>#!BuildConstraint: hardware:physicalmemory:size unit=M 1400</screen>
+    </sect4>
    </sect3>
   </sect2>
  </sect1>
- <sect1>
+ <sect1 xml:id="_constraint_handling">
   <title>Constraint Handling</title>
-  <para> The constraint handling depends on what is met by the restrictions.
-   The handling starts when there is no worker to fulfill the constraints. </para>
+  <para> What happens when someone sets a constraint so high, that the OBS
+   instance does not have even a single worker that meets it? What happens
+   when just a few workers satisfy all the constraints, but all of them are
+   busy building packages, or have been taken down for maintenance? This
+   section describes how the OBS handles these "low compliant worker"
+   situations. </para>
+  <sect2>
+   <title>At least one compliant worker is available</title>
+   <para> After determining which build workers satisfy the defined
+    constraints for a given build, the scheduler checks if any of them are
+    available to start building. If at least one is available, the build
+    begins. The rest of this section describes the OBS's behavior when no
+    compliant build workers are free to start building a given package. </para>
+  </sect2>
   <sect2>
-   <title>More than half of the workers satisfy the constraints</title>
-   <para> The job will just stay in state scheduled and no further notification
-    is set. </para>
+   <title>More than half of existing workers satisfy the constraints</title>
+   <para> The build will stay in state "scheduled" until one of the compliant
+    workers becomes available. No further notification is set. </para>
   </sect2>
   <sect2>
-   <title>Less than Half of the Workers Satisfy the Constraints</title>
-   <para> The job will stay in state scheduled and the dispatch details are set
-    to tell the user that this job can take a long time to be built. This will
+   <title>Less than half of existing workers satisfy the constraints</title>
+   <para> The build will stay in state scheduled until one of the compliant
+    workers becomes available. In addition, the dispatch details are set
+    to tell the user that this build might take a long time to complete. This will
     be shown in the Web UI on mouse over and the scheduled state will be
-    highlighted as well.</para>
```

Comment:
Maybe not specific mention the webui here, just saying the information is available in the detatils. This also available in api and osc (osc r -v)